src/test/resources/valid-example.json

```diff
@@ -5,10 +5,15 @@
         {
             "name": "20200122_1217_1-A1-B1-PAE12345_1234567a",
             "metadata":  {
-                "start_date": "20201105 11:15",
-                "flowcell_id": "PAE26306",
-                "instrument": "PCT0094",
-                "flowcell_position": "2-A3-D3"
+                "asic_temp": "32.631687",
+                "device_type" : "promethion",
+                "flow_cell_id": "PAE26306",
+                "flow_cell_product_code": "FLO-PRO002",
+                "flow_cell_position": "2-A3-D3",
+                "guppy_version":  "3.2.8+bd67289",
+                "hostname": "PCT0094",
+                "protocol": "sequencing/sequencing_PRO002_DNA:FLO-PRO002:SQK-LSK109:True",
+                "started": "2020-02-11T15:52:10.465982+01:00"
             },
             "path": "file:///Users/kocht/SoftwareDevelopment/libs/core-utils-lib/src/test/resources/dummyFileSystem/nanopore-instrument-output/validates/QABCD001AB_E12A345a01_PAE12345/20200122_1217_1-A1-B1-PAE12345_1234567a/",
```

Comment:
Would it be possible to replace the absolute filePath with a relative filePath starting from src?